CREDITS.md

```diff
@@ -115,6 +115,7 @@ People are sorted by name so please keep this order.
 * [Julien-Pierre Avérous](https://github.com/javerous): [contributions](https://github.com/FreshRSS/FreshRSS/pulls?q=is:pr+author:javerous), [Web](https://www.sourcemac.com/)
 * [Justin Tracey](https://github.com/jtracey): [contributions](https://github.com/FreshRSS/FreshRSS/pulls?q=is:pr+author:jtracey), [Web](https://unsuspicious.click)
 * [Kaibin Yang](https://github.com/SkyYkb): [contributions](https://github.com/FreshRSS/FreshRSS/pulls?q=is:pr+author:SkyYkb), [Web](https://kaibinyang.com/)
+* [Kaiwalya Koparkar](https://github.com/kaiwalyakoparkar): [contributions](https://github.com/FreshRSS/FreshRSS/pulls?q=is:pr+author:kaiwalyakoparkar), [Web](https://kaiwalyakoparkar.com/)
```

Comment:
```suggestion
```

Let's wait for some code contributions before putting you there, but help welcome ;-)

Reply:
Ah sure, I saw the contributing md file and added this :)